packages/core/src/network/network-peers.ts

```diff
@@ -15,6 +15,7 @@ const MAX_DELAY = 5 * 60 * 1000 // 5mins
 const BACKOFF_EXPONENT = 1.5
 const MAX_BACKOFF = MAX_DELAY / MIN_DELAY
 const UNKNOWN_Q = 0.2 // Default quality for nodes we don't know about.
+const OFFLINE_THRESHOLD = 0.5 // Nodes with quality below are considered dead
```

Comment:
This threshold looks like we should be able to easily configure it for testing and/or network configuration.

Reply:
added a test checks if we can detect offline nodes - the opposite seems to be quite massive